### ISSUE TITLE
Some addons do not work

### ISSUE THREAD

**Issue by CHJ85** [MetalChris/repository.matrix#2]
Please take a look at Discovery Channel. None of the channels work.
And playback doesn't work on The CW addon. And the NewsMax doesn't work either.
Oh, and the True Crime addon has a  few playback issues. It jumps all over the place during commercials.

**Comment by MetalChris** [comment]
I can probably fix NewsMax, but don't have time right now.  I'm not going to bother to look at the commercials in the True Crime add-on as it's a known issue and I don't think there's anything I can do about it anyway.  As long as the shows work, that's good enough.  Playback on CW works fine for me.  Discovery Channels is a mess.  They've been transitioning all of the various channel websites over the last few months and I probably won't be able to fix it.  I'll look at it though, but not until this weekend at the earliest.

**Comment by CHJ85** [comment]
Thanks. Would it be possible to fix True Crime Network by somehow bypassing the ads? I don't know if there could be something in the API or maybe you can somehow prevent the addon from playing videos that are shorter than 5 minutes.
I'm not a programmer, so I wouldn't know. I'm just throwing out ideas.
Because the thing about it is, the shows do not always work. The first time I watched an episode through your addon, the show would not play after the second commercial break. All the ad breaks would play and ignored the actual show.

**Comment by MetalChris** [comment]
I can't do anything about the commercials.  If I could get rid of them I'd have already done that.

Working on the NewsMax add-on now.  Almost done. Should have an update out tonight or tomorrow.

**Comment by CHJ85** [comment]
Awesome!
Thanks for your great work.
But maybe there is something you can do about the commercials. Not removing them, but you could add half a minute pre-buffer. Hopefully that'll prevent them from jumping all over the place.Or maybe I'm wrong. Again, I'm not a programmer.

**Comment by MetalChris** [comment]
There is **_nothing_** I can do about the commercials in the True Crime Network streams.

I've uploaded a fixed version of the NewsMax add-on and marked the Discovery Channels add-on as broken.  Discovery has changed all of their streams to mpeg-dash, and I cannot fix the current version of the add-on.  It will require a complete re-write from the ground up, and I do not have the time or inclination for that now.

**Comment by CHJ85** [comment]
Okay thank you.
